jumanji/environments/logic/minesweeper/viewer.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Optional, Sequence, Tuple
+
+import chex
+import jax.numpy as jnp
+import matplotlib
+from matplotlib import pyplot as plt
+
+import jumanji.environments
+from jumanji.environments.logic.minesweeper.constants import (
+    DEFAULT_COLOR_MAPPING,
+    UNEXPLORED_ID,
+)
+from jumanji.environments.logic.minesweeper.types import State
+from jumanji.environments.logic.minesweeper.utils import explored_mine
+from jumanji.viewer import Viewer
+
+
+class MinesweeperViewer(Viewer[State]):
+    def __init__(
+        self,
+        color_mapping: Optional[List[str]] = None,
+        num_rows: int = 10,
+        num_cols: int = 10,
```

Comment:
```suggestion
        num_rows: int = 10,
        num_cols: int = 10,
        color_mapping: Optional[List[str]] = None,
```
Does this ordering make more sense?

Reply:
OK